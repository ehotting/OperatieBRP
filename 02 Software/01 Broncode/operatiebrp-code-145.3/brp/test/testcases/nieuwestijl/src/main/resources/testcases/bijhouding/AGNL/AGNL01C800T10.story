Meta:
@auteur                 tjlee
@status                 Klaar
@sleutelwoorden         Klaar
@regels                 R1572
@usecase                UCS-BY.HG

Narrative:
R1572 Huwelijk en geregistreerd partnerschap, voltrekking huwelijk in Nederland, actie registratieAanvangHuwelijkGeregistreerdPartnerschap

Scenario: R1572 Ouder en wijzigen Geslachtsnaamcomponent.stam en Ouder.Ouderschap.DAG overlapt met Geslachtsnaamcomponent.Standaard.DAG
          LT: AGNL01C800T10

Given alle personen zijn verwijderd
Given enkel initiele vulling uit bestand /LO3PL-AGNL/AGNL01C800T10-Marjan.xls
Given enkel initiele vulling uit bestand /LO3PL-AGNL/AGNL01C800T10-Victor.xls

When voer een bijhouding uit AGNL01C800T10.xml namens partij 'Gemeente BRP 1'

Then heeft het antwoordbericht verwerking Geslaagd
And is het antwoordbericht gelijk aan /testcases/bijhouding/AGNL/expected/AGNL01C800T10.xml voor expressie //brp:bhg_hgpRegistreerHuwelijkGeregistreerdPartnerschap_R

Then is in de database de persoon met bsn 485927081 wel als PARTNER betrokken bij een GEREGISTREERD_PARTNERSCHAP
Then is in de database de persoon met bsn 188518113 wel als PARTNER betrokken bij een GEREGISTREERD_PARTNERSCHAP
