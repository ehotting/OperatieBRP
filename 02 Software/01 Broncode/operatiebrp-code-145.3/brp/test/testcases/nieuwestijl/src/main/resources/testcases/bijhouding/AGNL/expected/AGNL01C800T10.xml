<?xml version="1.0" encoding="UTF-8"?><brp:bhg_hgpRegistreerHuwelijkGeregistreerdPartnerschap_R xmlns:brp="http://www.bzk.nl/brp/brp0200">
   <brp:stuurgegevens>
      <brp:zendendePartij>199901</brp:zendendePartij>
      <brp:zendendeSysteem>BRP</brp:zendendeSysteem>
      <brp:referentienummer>*</brp:referentienummer>
      <brp:crossReferentienummer>00000000-0000-0000-0042-200000000104</brp:crossReferentienummer>
      <brp:tijdstipVerzending>*</brp:tijdstipVerzending>
   </brp:stuurgegevens>
   <brp:resultaat>
      <brp:verwerking>Geslaagd</brp:verwerking>
      <brp:bijhouding>Verwerkt</brp:bijhouding>
      <brp:hoogsteMeldingsniveau>Waarschuwing</brp:hoogsteMeldingsniveau>
   </brp:resultaat>
   <brp:meldingen>
      <brp:melding brp:objecttype="Melding" brp:referentieID="comid.aangaanGPInNL.registratieGeslachtsnaam.persoon">
         <brp:regelCode>R1572</brp:regelCode>
         <brp:soortNaam>Waarschuwing</brp:soortNaam>
         <brp:melding>*</brp:melding>
      </brp:melding>
   </brp:meldingen>
   <brp:aangaanGeregistreerdPartnerschapInNederland brp:objecttype="AdministratieveHandeling">
      <brp:partijCode>507013</brp:partijCode>
      <brp:tijdstipRegistratie>*</brp:tijdstipRegistratie>
      <brp:bijgehoudenPersonen>*</brp:bijgehoudenPersonen>
        <brp:bijhoudingsplan brp:objecttype="Bijhoudingsplan">*</brp:bijhoudingsplan>
   </brp:aangaanGeregistreerdPartnerschapInNederland>
</brp:bhg_hgpRegistreerHuwelijkGeregistreerdPartnerschap_R>
